core/services/relay/evm/evm.go

```diff
@@ -143,7 +143,22 @@ func NewRelayer(lggr logger.Logger, chain legacyevm.Chain, opts RelayerOpts) (*R
 		mercuryORM:           mercuryORM,
 		transmitterCfg:       opts.TransmitterConfig,
 		capabilitiesRegistry: opts.CapabilitiesRegistry,
-	}, nil
+	}
+
+	// Initialize write target capability if configuration is defined
+	if chain.Config().EVM().ChainWriter().ForwarderAddress() != nil {
```

Comment:
We should probably rename this to "KeystoneForwarderAddress" to not be confused with Operator Forwarders.